nltk/tag/perceptron.py

```diff
@@ -381,17 +361,19 @@ def _load_data_conll_format(filename):
         return sentences
 
 
-def _get_pretrain_model():
```

Comment:
FYI It's a convention used across NLTK to provide demonstration functionality when the module is called on the command line.

Reply:
Thanks @stevenbird, the added demo functionality seems quite useful.